src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$day-size: 100px;
+$gap-between-days: 1px;
+$calendar-side-paddings: 10px;
```

Comment:
I think you create too a lot of variables. For example variables for 1px)) And I don't understand calendar-side-paddings. Why 10px, you can use numbers from the day of the week.

Reply:
- I decided to add all these variables for the function that limit the width of the container to protect it correct work
- I thought the same way about 1px)) but decided to add it in case somebody will decide to make gap bigger + I used it in  several places
- maybe I didn't understand task correct, but I take 10px here:
![image](https://user-images.githubusercontent.com/87998640/221910293-40d9ba18-f034-4dc2-aff7-6d2ce41d97fc.png)